server/views/temporary-accommodation/assessments/show.njk

```diff
@@ -66,6 +69,38 @@
     {% endfor %}
   </div>
 
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <form action="{{ paths.assessments.notes.create({ id: assessment.id }) }}" method="post">
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+        {{
+        govukTextarea({
+          name: "note",
+          id: "note",
+          label: {
+            text: "Add a note",
+            classes: "govuk-label--m referral-note"
+          }
+        })
+        }}
```

Comment:
Suggest use taTextarea here instead, it'll save some repetition. And use the same naming as the field in the API. If this is "message" rather than "notes", everything will link up by magic (ie. `catchValidationErrorOrPropogate`) if there's an error from the API - this'll be be highlighted in red, and the errors at the top of the page will link to here

Reply:
:recycle: